utils/tools.js

```diff
@@ -1,116 +1,72 @@
-const nacl = require('tweetnacl')
-nacl.util = require('tweetnacl-util')
-const crypto = require('crypto')
-const ursa = require('ursa')
+const nacl = require('tweetnacl');
+nacl.util = require('tweetnacl-util');
+const crypto = require('crypto');
+const ursa = require('ursa');
+
+const d64 = data => nacl.util.decodeBase64(data);
+
+const e64 = data => nacl.util.encodeBase64(data);
+
+const dUTF = data => nacl.util.decodeUTF8(data);
+
+const eUTF = data => nacl.util.encodeUTF8(data);
 
 const keypair = () => {
-  var keys = ursa.generatePrivateKey(2048, 65537);
+  const keys = ursa.generatePrivateKey(2048, 65537);
 
-  var priv = keys.toPrivatePem('base64');
-  var pub = keys.toPublicPem('base64');
+  const priv = keys.toPrivatePem('base64');
+  const pub = keys.toPublicPem('base64');
 
-  return {pub, priv};
+  return { pub, priv };
 };
 
 const sha256 = (data) => {
   const hash = crypto.createHash('sha256');
-  hash.update(data)
-  return hash.digest('hex')
-}
+  hash.update(data);
+  return hash.digest('hex');
+};
 
 
 const uuid = () => {
   const ch = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ9';
-  var id = '';
-  for (var i = 0; i < 26; i++) {
+  let id = '';
+  for (let i = 0; i < 26; i++) {
     id += ch.charAt(Math.floor(Math.random() * ch.length));
   }
   return id;
 };
 
-const verify = async (msg, sig, issuerID) => {
-  var issuer = await Iota.getBundles(issuerID, 'I');
-  return eUTF(crypto.publicDecrypt(eUTF(d64(issuer[0].message.pk)), Buffer(d64(sig)))) === msg;
-};
-
-const verifyHash = async (root, sig, issuerID) => {
-  var issuer = await Iota.getBundles(issuerID, 'I');
-  return eUTF(crypto.publicDecrypt(eUTF(d64(issuer[0].message.pk)), Buffer(d64(sig)))) === sha256(root);
-};
-
-const sign = (msg, sk) => {
-  return e64(crypto.privateEncrypt(eUTF(d64(sk)), Buffer(dUTF(msg))));
-};
+const sign = (msg, sk) => e64(crypto.privateEncrypt(eUTF(d64(sk)), Buffer.from(dUTF(msg))));
 
-const signHash = (msg, sk) => {
-  return sha256(e64(crypto.privateEncrypt(eUTF(d64(sk)), Buffer(dUTF(msg)))));
-};
+const signHash = (msg, sk) => sha256(e64(crypto.privateEncrypt(eUTF(d64(sk)), Buffer.from(dUTF(msg))))); // eslint-disable-line max-len
 // Encrypt data for transmit
-const encrypt = (msg, pk) => {
-  return e64(crypto.publicEncrypt(eUTF(d64(pk)), Buffer(dUTF(msg))));
-};
+const encrypt = (msg, pk) => e64(crypto.publicEncrypt(eUTF(d64(pk)), Buffer.from(dUTF(msg))));
 
 // Decrypt data
-const decryptUTF = (msg, sk) => {
```

Comment:
To me, it is NOT trivial to interpret these lines as "style tweaks".

Reply:
This is Airbnb coding style use arrow-body-style rule with `as-needed` option, so I change these line to fit the coding style.

If it is inappropriately, maybe we could use this rule with `always`. 🤔 

See the different in this [link](https://eslint.org/docs/rules/arrow-body-style).

Reply:
I defer to `STYLE-GUIDE.md` and you shall ensure most style tweaks can be automated without human interactions.

Reply:
This coding style can be automatically fixed by the `eslint -fix`.
[List of available rules](https://eslint.org/docs/rules/)